src/QueuedNotification.php

```diff
@@ -500,6 +500,11 @@ public static function cronInfo($name)
                 return ['description' => __('Clean notification queue'),
                     'parameter'   => __('Days to keep sent emails')
                 ];
+
+            case 'queuednotificationcleanunset':
```

Comment:
1. "unset" had a typo, and "stale" is probably more appropriate.
2. "ajax" term should be added to be precise enough.
```suggestion
            case 'queuednotificationcleanstaleajax':
```